docs/guides/implementation-guides/how-to-implement-dependency-injection.md

```diff
@@ -100,6 +100,8 @@ public partial class App : Application
         // Without this line you will get duplicate validations from both Avalonia and CT
         BindingPlugins.DataValidators.RemoveAt(0);
 
+        // `ServiceManager` retrieve an instance of `MainViewModel`
```

Comment:
I would reword this to be far clearer so someone who has never used DI before can see more clearly what it does. Something like "The service manager's locator instance is used to resolve an instance of MainViewModel, with all the constructor depedencies of MainViewModel such as MyService that was configured during the container creation."

But there may be a better way of phrasing all of that.